core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -523,6 +530,19 @@ public static int defaultMaxTotalAttempts() {
         return COMPOSITE_SERVICE_CACHE_SPEC;
     }
 
+    /**
+     * Returns the value of the {@code annotatedServiceExceptionLoggingMode} parameter which equals to
```

Comment:
Could you rephrase this to better explain what this flag does and why it is necessary?